nyaa/templates/profile.html

```diff
@@ -6,16 +6,16 @@
 <h2 style="margin-bottom: 20px;">Profile of <strong class="text-{{ g.user.userlevel_color }}">{{ g.user.username }}</strong></h2>
 
 <div class="row" style="margin-bottom: 20px;">
-	<div class="col-sm-2" style="max-width: 150px;">
-		<img class="avatar" src="{{ g.user.gravatar_url() }}">
-	</div>
-	<div class="col-sm-10">
-		<dl class="row" style="margin: 20px 0 15px 0;">
-			<dt class="col-sm-2">User ID:</dt><dd class="col-sm-10">{{ g.user.id }}</dd>
-			<dt class="col-sm-2">User Class:</dt><dd class="col-sm-10">{{ g.user.userlevel_str }}</dd>
-			<dt class="col-sm-2">User Created on:</dt><dd class="col-sm-10">{{ g.user.created_time }}</dd>
-		</dl>
-	</div>
+    <div class="col-sm-2" style="max-width: 150px;">
```

Comment:
Our style is to use tabs as indentation in template files, presumably so people can change their tab width to get a better view of deeply indented content. You've used spaces here.